Src/IronPython/Runtime/Operations/StringOps.cs

```diff
@@ -1939,6 +1939,23 @@ private static class CodecsInfo {
 #endif
                 return d;
             }
+
+            internal static readonly Lazy<Dictionary<string, object>> ErrorHandlers = new Lazy<Dictionary<string, object>>(MakeErrorHandlersDict);
+
+            private delegate object ErrorHandler(object unicodeError);
+
+            private static Dictionary<string, object> MakeErrorHandlersDict() {
+                var d = new Dictionary<string, object>();
+
+                d["strict"] = new ErrorHandler(StrictErrors);
```

Comment:
You can wrap the method in `BuiltinFunction` instead of creating delegates. Something like the following:
```C#
d["strict"] = BuiltinFunction.MakeFunction("strict_errors", ReflectionUtils.GetMethodInfos(typeof(StringOps).GetMember(nameof(StrictErrors), BindingFlags.Static | BindingFlags.NonPublic)), typeof(StringOps));
```
Note that I'm super familiar with this part of the codebase (`BuiltinFunction`) so I may be misusing it (e.g. what is the proper `declaringType`).